### PR TITLE
Drop ldapmp module

### PR DESCRIPTION
**Pull request by gforcada**
To be merged after #71 is merged 😄

### CODE REVIEW

**Comment by jensens** [comment]
However, this contains  a breaking change and should result in a major version bump.

**Comment by gforcada** [comment]
I'm always shy to add breaking news snippets 😊 but you are right 👍🏾

**Comment by jensens** [comment]
And I am almost sure plone.openid is obsolete as well.

**Comment by gforcada** [comment]
We can drop that one afterwards

**Comment by jensens** [comment]
> We can drop that one afterwards

but do not add it to setup.py. It is already optional (condition import) and it is unsupported dead code.

File: setup.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "plone.base",
     "plone.i18n",
     "plone.memoize",
+    "plone.openid",
```
Comment (jensens) [inline setup.py:19]: Do not add this.
```suggestion
```

Comment (gforcada) [reply setup.py:19]: Oh, sorry 🤦🏾 I've been configuring so many projects that I don't remember what I did on each of them 😆 

Will do, thanks!

**Pull request merged**